api-server-nest/src/app.module.ts

```diff
@@ -0,0 +1,27 @@
+import {Module} from '@nestjs/common';
+import {VenuesModule} from "./venues/venues.module";
+import {SequelizeModule} from "@nestjs/sequelize";
+import {CurrentEventsModule} from "./current-events/current-events.module";
+import {AppController} from "./app.controller";
+
+require('dotenv').config();
+
+@Module({
+    imports: [
+        SequelizeModule.forRoot({
+            dialect: 'postgres',
+            autoLoadModels: true,
+            synchronize: true,
+            host: process.env.DB_HOST,
+            port: +process.env.DB_PORT,
```

Comment:
Is the `+` here to trigger a cast to `Number`? If so, seems like `parseInt` would be clearer.